generators/client/templates/src/main/webapp/app/admin/health/_health.modal.controller.js

```diff
@@ -17,5 +17,14 @@
         function cancel() {
             $uibModalInstance.dismiss('cancel');
         }
+
```

Comment:
Oh sorry @deepu105 I should have introduced you to @040796 (who definitely has a strange username) - he's my new trainee, I'm going to have a look at this with him